eox_tenant/signals.py

```diff
@@ -174,30 +197,12 @@ def start_tenant(sender, environ, **kwargs):  # pylint: disable=unused-argument
 
     domain = http_host.split(':')[0]
 
-    # Find what we need to do about the current setting and the incoming request.domain
-    must_reset, can_keep = _analyze_current_settings(domain)
-
-    # Reset minimum once every so often
-    must_reset = _ttl_reached() or must_reset
-
-    # Perform the reset
-    if must_reset:
-        _perform_reset()
-        can_keep = False
-
-    if can_keep:
+    if reset_or_keep_settings(domain):
```

Comment:
Ironically the `reset_or_keep_settings` function returns a Boolean for an answer? instead of the string `reset` or `keep`. I would say the function should be called "can_keep_settings?" because then the True/False value makes more sense